packages/backend/src/graphql/mutations/update-flow-status.ts

```diff
@@ -43,24 +42,10 @@ const updateFlowStatus = async (
     pattern: interval || EVERY_15_MINUTES_CRON,
   }
 
-  if (trigger.type === 'webhook') {
-    const $ = await globalVariable({
-      flow,
-      connection: await triggerStep.$relatedQuery('connection'),
-      app: await triggerStep.getApp(),
-      step: triggerStep,
-      testRun: false,
-    })
-
-    if (flow.active && trigger.registerHook) {
```

Comment:
nit: can we remove `registerHook` and `unregisterHook` from `IBaseTrigger` declaration as well

Reply:
nvm I see we're still using `registerHook` in later PR